DSLink/Util/TimeUtil.cs

```diff
@@ -2,9 +2,9 @@
 
 namespace DSLink.Util
 {
-    internal static class TimeUtil
+    public static class TimeUtil
     {
-        public static string ToIso8601(DateTime dateTime)
+        public static string ToIso8601(this DateTime dateTime)
```

Comment:
It's just an extension method, allows you to do DateTime.ToIso8601, instead of calling TimeUtil.ToIso8601(dateTime).